index.html

```diff
@@ -0,0 +1,288 @@
+                        <li><div id="work4tag3"><div id="javascript4">javascript</div></div></li>
+                    </ul>
+                </div>
+                <div class="work4Actions">
+                    <button id="work4see-project">
+                       <b>See Project</b>
+                    </button>
+                </div>
+               </div>
+               
+            </section>
+        </section>
+        <section class="About-myself">
+            <section class="top">
+                <div id="primary-heading"><h2>About Myself</h2></div>
+                <div id="primary-paragraph">
+                    <p>Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="lets-connect">LET’S CONNECT</div>
+                    <div class="connect-icons">
+                        <ul>
+                            <li><a href="https://github.com/Kevo-hacker"><img src="Vector2.png" alt=""></a></li>
+                            <li><a href="https://www.linkedin.com/in/kelvin-ben-323043173/"><img src="Linkedin.png" alt=""></a></li>
+                            <li><a href="peace.com"><img src="Vector3.png" alt=""></a></li>
+                            <li><a href="https://twitter.com/kevoyout144"><img src="Vector.png" alt=""></a></li>
+                            <li><a href="Gmail.com"><img src="Vector1.png" alt=""></a></li>
+                        </ul>
+                    </div>
+                     <button id="my-resume"><span>Get my resume</span></button>
+            </section>
+            <div class="skill-list">
+                    <div id="frame12">
+                        <div id="language">
+                            <div id="drop-down">
+                                <img src="Chevron-Down.png" alt=""></div>
+                                <div id="lang">
+                                    <h5>Languages</h5>
+                                </div>
+                            
+                        </div>
+                    </div>
+                    <div id="lang-list">
+                        <div id="frame9">
+                            <div id="img"><img src="Ellipse1.png" alt=""></div>
+                            <div id="js">javascript</div>
+                            
+                        </div>
+                    </div>
+                    <div id="frame10">
+                        <div id="img1"><img src="Ellipse2.png" alt=""></div>
+                            <div id="html">HTML</div>
+                    </div>
+                    <div id="frame11">
+                        <div id="img2"><img src="Ellipse0.png" alt=""></div>
+                            <div id="css">CSS</div>
```

Comment:
- [ ] You still have the same problem here too. Please, ensure that you have all the images you need in a folder and you use the correct image path to display them. I know can do it. Courage! 👏 😊